src/tools/fuzzing.h

```diff
@@ -177,6 +177,7 @@ class TranslateToFuzzReader {
 
   // Optionally remove NaNs, which are a source of nondeterminism (which makes
   // cross-VM comparisons harder)
+  // TODO: de-NaN SIMD values
```

Comment:
@kripken how critical is this given that the default setting is to enable SIMD values in the fuzzer?

Reply:
Well, NaNs can cause some nondeterminism, which makes comparing between VMs hard. But definitely something we can leave for later.